cpp/vm/evmzero/interpreter_test.cc

```diff
@@ -156,6 +156,14 @@ void RunInterpreterStepTest(const InterpreterTestDescription& desc, int steps =
   EXPECT_EQ(ctx.return_data, desc.return_data);
 }
 
+Stack& fullStack() {
+  static auto stack = Stack();
```

Comment:
Is this an optimization? to avoid generating new ones?
We will flash it clean with zeroes anyway. I guess a new instance would be safer.

Reply:
The stack overflow tests will be implemented in another PR since it requires a little more work as discussed with @LuisPH3